src/routes/articles/delete-article.e2e.test.js

```diff
@@ -0,0 +1,82 @@
+const {expect} = require('chai')
+const {fixtures} = require('../../../test/utils')
+
+
+describe('DELETE /articles/:id', () => {
+	let user
+	let accessToken
+	before(async ()=>{
+		 user = await fixtures.insertUser() 
+		const body = { id: user.id, email: user.email }
+		accessToken = fixtures.generateAccessToken(
+			{user : body}
+		)
+	})
+	describe('Failure', () => {
+		
+		it('should return 400 if postId is not a number', async () => {
+			const expectedError = {
+				"error": {
+			  "message": "id must be a number"
+				},
+		   "status": "failed"
+		  }
+			return	fixtures.api()
+				.delete(`/api/v1/articles/sdsdsd`)
+				.set('Authorization', `Bearer ${accessToken}`)
+				.expect(400, expectedError)
+
+		})
+
+		it('should return 401 if request is not authenticated', async () => {
+			const post = await fixtures.insertPost({
+				userId : user.id , 
+				type : 'article'
+			}) 
+			return fixtures.api()
+				.delete(`/api/v1/articles/${post.id}`)
+				.expect(401)
+
+		})
+	})
+
+	describe('Success', () => {
+
+		it('should return 200', async () => {
+			const post = await fixtures.insertPost({
+				userId : user.id , 
+				type : 'article'
+			})
+			await fixtures.insertPostFlag({
+				userId : user.id , 
+				postId : post.id}) 
+			return fixtures.api()
+				.delete(`/api/v1/articles/${post.id}/flags/${user.id}`)
```

Comment:
kindly check this. Not sure this is endpoint we're testing

Reply:
an oversight, thanks a lot

Reply:
i have pushed the changes